### PR TITLE
Fix window resizing bug

### PR DESCRIPTION
**Pull request by 4z0t**

### CODE REVIEW

**Comment by Garanas** [comment]
What was the bug exactly?

File: lua/maui/window.lua [hunk 1]
```diff
@@ -485,8 +484,8 @@ Window = Class(Group) {
                 self.Top:Set(top)
 
                 -- we can scale these accordingly as we applied the inverse on saving
-                self.Width:Set(LayoutHelpers.ScaleNumber(width))
-                self.Height:Set(LayoutHelpers.ScaleNumber(height))
+                self.Right:Set(LayoutHelpers.ScaleNumber(width) + left)
```
Comment (speed2CZ) [inline lua/maui/window.lua:487]: this looks like smth that would break with scaling

**Comment by 4z0t** [comment]
On current dev branch, reaize chat window and move it, you will see the bug

File: lua/maui/window.lua (hunk 1 above)
Comment (4z0t) [reply lua/maui/window.lua:487]: Wdym?

**Comment by 4z0t** [comment]
It also happens with other windows, like minimap

**Comment by Garanas** [comment]
interesting, I'll review it 😄

**Comment by 4z0t** [comment]
Basically the problem was in using Height and Width as reference for window size, it means  Bottom and Right are dependent on Top and Height and Left and Width, but resizing of window changes Top, Left, Bottom and Right causing window having 6 components being independent of each other.

**Comment by 4z0t** [comment]
@Garanas will you merge this?

**Pull request merged**